### PR TITLE
[ADD] sale_auto_remove_zero_quantity_lines

### PR DESCRIPTION
**Pull request by santostelmo**
Upon confirming a sales order remove any sales order lines that have zero quantities.

### CODE REVIEW

File: sale_auto_remove_zero_quantity_lines/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    _inherit = "sale.order"
+
+    def action_confirm(self):
+        for order in self:
+            if order.company_id.sale_auto_remove_zero_quantity_lines:
+                zero_lines = order.order_line.filtered(
+                    lambda line: line.product_uom_qty == 0
+                )
+                if zero_lines:
+                    body = _(
```
Comment (rousseldenis) [inline sale_auto_remove_zero_quantity_lines/models/sale_order.py:17]: I would have logged the lines names and quantities

**Review by rousseldenis — COMMENTED** [review]
LGTM. A little change to better log removed lines

**Review by cyrilmanuel — APPROVED** [review]
LGTM good job

File: sale_auto_remove_zero_quantity_lines/models/sale_order.py (hunk 1 above)
Comment (santostelmo) [reply sale_auto_remove_zero_quantity_lines/models/sale_order.py:17]: In our requirements we prefer to not have this level of details being logged in the form.

**Review by AungKoKoLin1997 — APPROVED** [review]
LGTM.

**Pull request merged**